src/flynt/process.py

```diff
@@ -152,11 +152,11 @@ def maybe_replace(self, chunk, contract_lines, converted, rest):
             self.last_idx += 1
 
     def add_rest(self):
-        self.results.append(self.src_lines[self.last_line][self.last_idx :] + "\n")
+        self.results.append(f"{self.src_lines[self.last_line][self.last_idx:]}\n")
         self.last_line += 1
 
         while len(self.src_lines) > self.last_line:
-            self.results.append(self.src_lines[self.last_line] + "\n")
```

Comment:
while it might(?) increase performance, didn't it reduce readability? I think it's harder to see what is being added, what do you think

Reply:
You can tell something is being added from the fact that it's in an f-string, and when you scan to the end of the line, you see it's an `"\n"`. I think it's fine.

Reply:
Yes, lets revert the ones in process.py. For 2 strings its even slower. and I find old version more readable. https://stackoverflow.com/questions/59180574/string-concatenation-with-vs-f-string